src/core/Akka.Remote.TestKit/Conductor.cs

```diff
@@ -65,17 +66,23 @@ public IActorRef Controller
         public async Task<INode> StartController(int participants, RoleName name, INode controllerPort)
         {
             if(_controller != null) throw new IllegalStateException("TestConductorServer was already started");
-            _controller = _system.ActorOf(new Props(typeof (Controller), new object[] {participants, controllerPort}),
-                "controller");
+            _controller = _system.ActorOf(Props.Create(() => new Controller(participants, controllerPort)),
+               "controller");
             //TODO: Need to review this async stuff
-            var node = await _controller.Ask<INode>(TestKit.Controller.GetSockAddr.Instance).ConfigureAwait(false);
+            var node = await _controller.Ask<INode>(TestKit.Controller.GetSockAddr.Instance, Settings.QueryTimeout).ConfigureAwait(false);
```

Comment:
I made a lot of the same changes everywhere - every `Ask` operation used by the internals of the MultiNodeTestkit (such as the `Player`, `Controller`, `Conductor`, etc...) all have a configurable timeout.